gradio/queue.py

```diff
@@ -211,70 +240,91 @@ def get_estimation(self) -> Estimation:
             queue_eta=self.queue_duration,
         )
 
-    async def call_prediction(self, event: Event):
+    async def call_prediction(self, events: List[Event], batch: bool):
+        data = events[0].data
+        if batch:
+            data["data"] = [event.data["data"] for event in events]
```

Comment:
nit: but my editor is complaining about the types here. Event.data is supposed to be `PredictBody` but it's treated as a dict.

Reply:
Fixed, thanks